doc/api.md

```diff
@@ -238,6 +238,8 @@ changes. The listener will receive an object with the following structure:
 
 * **readyToClose** - event notification fired when Jitsi Meet is ready to be closed (hangup operations are completed).
 
+* **uiLoaded** - event notification fired when Jitsi Meet UI is ready with loading.
```

Comment:
Is the timing on this really when the UI is loaded or when resources have been loaded? UI loaded to me sounds like the UI has completed and is useable.

Reply:
Well, by UI I mean we have dom and we have css loaded, so we for sure have some view :)